spec/acceptance/suites/default/50_grafana_spec.rb

```
@@ -363,13 +364,6 @@ class { 'simp_grafana':
     it 'is idempotent' do
       apply_manifest_on grafana, manifest, :catch_changes => true
     end
-
```

The original author put this test in for a reason.  Is it no longer needed?

---

It's been a while, but I think the test was added here out of convenience-- due to time-constraints on the initial work.  It can probably be moved to [simp_elasticsearch](/simp/pupmod-simp-simp_elasticsearch).  On the other hand, I might have added it as a sanity check to fail early in case things weren't all fine and dandy.  The full test suite took around 45minutes, IIRC, so identifying failure very early was a high priority.

---

@lnemsick-simp @lamawithonel We can move this test to simp_elasticsearch if needed, but this shouldn't be a test for the simp_grafana module. Grafana and Elasticsearch have both changed their API and the goal of this module is not to verify if grafanas internal proxy from the front end to the back end is working correctly.

---

@brandonrdn Do you not need it as a sanity check, an [assertion](http://wiki.c2.com/?WhatAreAssertions)?